icu4c/source/i18n/number_mapper.cpp

```diff
@@ -127,8 +127,7 @@ MacroProps NumberPropertyMapper::oldToNew(const DecimalFormatProperties& propert
     // For backwards compatibility, minimum overrides maximum if the two conflict.
     // The following logic ensures that there is always a minimum of at least one digit.
```

Comment:
Please delete this line because this comment is no longer true.

Reply:
OK, removed